sunpy/visualization/animator/line.py

```diff
@@ -156,4 +156,4 @@ def update_plot(self, val, line, slider):
                 line.set_xdata(self.xdata[tuple(item)])
             slider.cval = val
         # Update slider label to reflect real world values in axis_ranges.
-        super().update_plot(val, slider)
+        super().update_plot(val, None, slider)
```

Comment:
I think it makes more sense to pass `line` here and not `None`

Reply:
If I remember right, this causes some issue. Let me try.

Reply:
Okay, right it works (obviously). My bad.